var/da/da_obs_io/da_read_obs_radar.inc

```diff
@@ -219,7 +219,7 @@ subroutine da_read_obs_radar (iv, filename, grid)
          endif
 
          call da_llxy (platform%info, platform%loc, outside, outside_all)
-         if( outside_all ) then
+         if( outside_all .and. multi_inc == 0 ) then
```

Comment:
@jamiebresch You or I added this one line change? No same change for other obs types. Looks like other obs types do not write out QC and obs error info, but your more efficient binary radar I/O writes out QC and obs error info in observer mode. Is that the reason for this one line change?